packages/trip-viewer-overlay/src/TripViewerOverlay.story.tsx

```diff
@@ -1,37 +1,30 @@
-import BaseMap, { Styled as BaseMapStyled } from "@opentripplanner/base-map";
 import React from "react";
 
 import tripData from "../__mocks__/mock-trip.json";
 import TripViewerOverlay from ".";
+import { withMap } from "../../../.storybook/base-map-wrapper";
```

Comment:
I have not checked the other stories, but the `import ... from "."` should be below this one.